### PR TITLE
Add pod-density workload

### PR DESCRIPTION
**Pull request by smalleni**
Signed-off-by: Sai Sindhur Malleni <smalleni@redhat.com>

### CODE REVIEW

**Comment by smalleni** [comment]
Depends on: https://github.com/cloud-bulldozer/benchmark-operator/pull/528

**Comment by Harshith-umesh** [comment]
Can one of the admins verify this patch?

**Review by rsevilla87 — APPROVED** [review]
LGTM!

File: workloads/kube-burner/run_poddensity_test_fromgit.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/bash -e
+
+set -e
+
+export WORKLOAD=pod-density
+export METRICS_PROFILE=${METRICS_PROFILE:-metrics.yaml}
+export JOB_ITERATIONS=${JOB_ITERATIONS:-1000}
```
Comment (rsevilla87) [inline workloads/kube-burner/run_poddensity_test_fromgit.sh:7]: This should be:
```
export JOB_ITERATIONS=${PODS:-1000} 
```

right?

Can you add this value to env.sh and reference its default value in README.md?

Comment (smalleni) [reply workloads/kube-burner/run_poddensity_test_fromgit.sh]: Fixed

**Comment by smalleni** [comment]
@rsevilla87 all fixed.

**Comment by rsevilla87** [comment]
> @rsevilla87 all fixed.

There're some pending changes in the benchmark-operator's pr ;)

**Review by rsevilla87 — APPROVED** [review]
LGTM

**Pull request merged**